### PR TITLE
Streamline SPM imports

### PR DESCRIPTION
**Pull request by scannillo**
### Summary

These changes will help streamline our docs, as well as simplify things for merchants.

Before:
- 3DS via SPM: Include the `BraintreeThreeDSecure` and `PayPalDataCollector` frameworks
- PayPal via SPM: Include the `BraintreePayPal` and `PayPalDataCollector` frameworks
- Local Payments via SPM:  Include the `BraintreePaymentFlow` and `PayPalDataCollector` frameworks

After:
- 3DS via SPM: Include the `BraintreeThreeDSecure` framework
- PayPal via SPM: Include the `BraintreePayPal` framework
- Local Payments via SPM:  Include the `BraintreePaymentFlow` framework

### Changes

- Add `PPRiskMagnes` binary as target to SPM libraries which require PayPalDataCollector.

### Checklist

- [X] Added a changelog entry

### Authors
@scannillo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## unreleased
 * Re-organize `/Frameworks` binaries into nested `/FatFrameworks` and `/XCFrameworks` directories.
   * Provides fix for this [CocoaPods issue](https://github.com/CocoaPods/CocoaPods/issues/10731) & allows proper usage of `PPRiskMagnes.xcframework` by `PayPalDataCollector` subspec.
+* Swift Package Manager
+  * Update Package.swift to include `PPRiskMagnes` as explicit target for library products that require `PayPalDataCollector`
+  * _Note:_ This simplifies integrations by no longer requiring manual inclusion of `PayPalDataCollector` to use `BraintreeThreeDSecure`, `BraintreePayPal`, and `BraintreePaymentFlow`
```
Comment (scannillo) [inline CHANGELOG.md:8]: What do y'all think of this CHANGELOG entry?

Comment (scannillo) [inline CHANGELOG.md:8]:
```suggestion
* Swift Package Manager
  * Update Package.swift to include `PPRiskMagnes` as explicit target for library products that require `PayPalDataCollector`
  * _Note:_ This simplifies integrations by no longer requiring manual inclusion of `PayPalDataCollector` in order to use `BraintreeThreeDSecure`, `BraintreePayPal`, and `BraintreePaymentFlow`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 ## unreleased
 * Re-organize `/Frameworks` binaries into nested `/FatFrameworks` and `/XCFrameworks` directories.
   * Provides fix for this [CocoaPods issue](https://github.com/CocoaPods/CocoaPods/issues/10731) & allows proper usage of `PPRiskMagnes.xcframework` by `PayPalDataCollector` subspec.
+* Swift Package Manager
+  * Update Package.swift to include `PPRiskMagnes` as explicit target for library products that require `PayPalDataCollector`
+  * _Note:_ This simplifies integrations by no longer requiring manual inclusion of `PayPalDataCollector` in order to use `BraintreeThreeDSecure`, `BraintreePayPal`, and `BraintreePaymentFlow`
```
Comment (scannillo) [inline CHANGELOG.md:8]: Thoughts on this CHANGELOG entry?

**Comment by scannillo** [comment]
UI Tests still flakey 😢 . We should investigate snapshot testing probably. I don't have a ton more ideas why this is happening, it only started happening with Xcode 13 IIRC.

File: CHANGELOG.md (hunk 2 above)
Comment (sarahkoop) [reply CHANGELOG.md:8]: CHANGELOG looks fine to me. Could simplify the second bullet point to just `No longer require manual inclusion...`

**Review by sarahkoop — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (scannillo) [reply CHANGELOG.md:8]:
```suggestion
* Swift Package Manager
  * Update Package.swift to include `PPRiskMagnes` as explicit target for library products that require `PayPalDataCollector`
  * _Note:_ No longer requires manual inclusion of `PayPalDataCollector` in order to use `BraintreeThreeDSecure`, `BraintreePayPal`, and `BraintreePaymentFlow`
```

I like it!

**Review by sshropshire — APPROVED** [review]
🌊

**Pull request merged**